composites/SnippetPreview/components/ProgressBar.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import colors from "../../../style-guide/colors.json";
+
+const ProgressBarContainer = styled.progress`
+	box-sizing: border-box;
+	width: 100%;
+	height: 8px;
+	display: block;
+	margin-top: 5px;
+	border: none;
+	-webkit-appearance: none;
+	appearance: none;
+	
+	::-webkit-progress-bar {
+	   	background-color: ${colors.$color_background_light};
+	   	color: ${( { color } ) => color };
+		border: 1px solid ${colors.$color_input_border};
+	}
+	
+	::-webkit-progress-value {
+		background-color: ${ props => props.progressColor };
+		transition: width 250ms;
+	}
+`;
```

Comment:
Seems the styling of the progress bar will work on webkit browsers only? We should support all browsers. Please refer to YoastSEO.js `_snippet-editor.scss` where there are some rules and a `@mixin` for the progress bar.